http-server/src/main/java/org/triplea/server/error/reporting/CreateIssueStrategy.java

```diff
@@ -10,6 +10,7 @@
 import org.triplea.http.client.error.report.ErrorUploadResponse;
 import org.triplea.http.client.github.issues.GithubIssueClient;
 import org.triplea.http.client.github.issues.create.CreateIssueResponse;
+import org.triplea.lobby.server.db.ErrorReportingDao;
```

Comment:
ErrorReportingDao was moved to the `lobby-db-dao` subproject, hence `http-server` subproject needs to import the dao